SPEC.md

```diff
@@ -5286,6 +5295,97 @@ Example output:
 </p>
 </details>
 
+### Runtime Access to Requirements, Hints, and Metadata
```

Comment:
Should we put in a recommendation that if these values are used in a way that significantly modifies the behavior of a task then it may lead to a workflow that will have reduced portability?

Reply:
I'm good with that. But perhaps you can give some examples of what types of usages you're worried about?

Reply:
We should just not allow it right? And force that behavior into hints?

Reply:
The metadata sections are free-form, but they should definitely not be used for hints, especially after we introduce a dedicated `hints` section. That said, I'm not sure we can force the Cromwell folks to do/not do anything, but we can strongly recommend against it in the spec.

Reply:
Agreed, we don't want this to become a way to "officially" implement what have to date been workarounds or engine specific optimizations